src/transformStateWithClones.js

```diff
@@ -6,8 +6,44 @@
  *
  * @return {Object[]}
  */
+function removeProperties(obj, toRemoveArray) {
```

Comment:
Let's create clear and descriptive variable names
`object` and `properties` will be great examples here
For now it's not clear what is this array, and what it should contain
Looks like it's some array that should be removed